src/components/AudioInfo.vue

```diff
@@ -523,128 +340,119 @@ const supportedFeatures = computed(
         .supportedFeatures) as EngineManifest["supportedFeatures"] | undefined
 );
 
-const setAudioSpeedScale = (speedScale: number) => {
-  store.dispatch("COMMAND_SET_AUDIO_SPEED_SCALE", {
-    audioKey: props.activeAudioKey,
-    speedScale,
-  });
-};
-
-const setAudioPitchScale = (pitchScale: number) => {
-  store.dispatch("COMMAND_SET_AUDIO_PITCH_SCALE", {
-    audioKey: props.activeAudioKey,
-    pitchScale,
-  });
-};
-
-const setAudioIntonationScale = (intonationScale: number) => {
-  store.dispatch("COMMAND_SET_AUDIO_INTONATION_SCALE", {
-    audioKey: props.activeAudioKey,
-    intonationScale,
-  });
-};
-
-const setAudioVolumeScale = (volumeScale: number) => {
-  store.dispatch("COMMAND_SET_AUDIO_VOLUME_SCALE", {
-    audioKey: props.activeAudioKey,
-    volumeScale,
-  });
-};
-
-const setAudioPrePhonemeLength = (prePhonemeLength: number) => {
-  store.dispatch("COMMAND_SET_AUDIO_PRE_PHONEME_LENGTH", {
-    audioKey: props.activeAudioKey,
-    prePhonemeLength,
-  });
-};
-
-const setAudioPostPhonemeLength = (postPhonemeLength: number) => {
-  store.dispatch("COMMAND_SET_AUDIO_POST_PHONEME_LENGTH", {
-    audioKey: props.activeAudioKey,
-    postPhonemeLength,
-  });
+type Parameter = {
+  label: string;
+  slider: PreviewSliderHelper;
+  action: Parameters<typeof store.dispatch>[0]["type"];
+  key: keyof Preset;
```

Comment:
nameとかもあるので、[こちら](https://github.com/VOICEVOX/voicevox/blob/6afc671ba4c4c267252bcc3a148bbce061090a0e/src/components/AudioInfo.vue#L811-L814C8)と合わせてこうしとくのどうでしょ。（あんま変わらないけど。。）
```suggestion
  key: keyof Omit<Preset, "name" | "morphingInfo">;
```